mcc/short/short/short.go

```diff
@@ -0,0 +1,121 @@
+package short
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+// NewService returns an empty Service
+func NewService() *Service {
+	return &Service{}
+}
+
+// Service holds identification and back-end methods
+type Service struct {
+	aPIKey            string
```

Comment:
I think unexported fields with acronyms should keep the whole acronym in lowercase, so it should be `apiKey`.